src/templates/en/2022/base.html

```diff
@@ -0,0 +1,10 @@
+{% extends "%s/base.html" % lang %}
+
+{% block methodology_stat_1 %}8.36M{% endblock %}
+{% block methodology_stat_2 %}43.88 TB{% endblock %}
+{% block total_websites %}nearly 8.4 million{% endblock %}
```

Comment:
@rviscomi any thoughts on secondary pages for these stats? At the moment they aren't included.

If I include them it goes up to 81.47 TB

Reply:
Let's omit their impact for now and update if they become a significant part of this edition